skactiveml/utils/_visualisation.py

```diff
@@ -0,0 +1,170 @@
+import warnings
```

Comment:
@LukasLuehrs Move helper functions (except for the check_bound) to the visualization package. Possible filename: `skactiveml.visualization._auxiliary_functions.py`.